PopPUNK/web.py

```diff
@@ -143,24 +143,39 @@ def calc_prevalence(cluster, cluster_list, num_samples):
     prevalence = round(clusterCount / num_samples * 100, 2)
     return prevalence
 
-def summarise_clusters(output, ref_db):
-    """Retreieve assigned query and all cluster prevalences"""
-    queryDF = pd.read_csv(os.path.join(output, os.path.basename(output) + "_clusters.csv"))
-    queryDF = queryDF.loc[queryDF['Taxon'] == "query"]
+def get_aliases(aliasDF, clusterLabels, species):
+    if species == 'Streptococcus pneumoniae':
+        GPS_name = 'unrecognised'
+        for label in clusterLabels:
+            if label in list(aliasDF['sample']):
+                index = list(aliasDF['sample']).index(label)
+                GPS_name = aliasDF['GPSC'][index]
+        alias_dict = {"GPS cluster":str(GPS_name)}
```

Comment:
Can you change the names on the results page? 'GPS cluster' to 'GPSC', and the poppunk cluster to 'strain' (which might be in the web code rather than here)